docs/dictionary/property/currentCard.lcdoc

```diff
@@ -0,0 +1,36 @@
+Name: currentCard
+
+Type: property
+
+Syntax: the currentCard
+
+Summary:
+A stack property that is set to the short name of the card in a given stack that is selected for display.
```

Comment:
This should probably just be `The short name of the <current card> of a stack`, linking to the current card glossary entry.